build.rs

```diff
@@ -0,0 +1,9 @@
+use rustc_version::{version_meta, Channel};
+
+fn main() {
+    // Set cfg flags depending on release channel
+    if let Channel::Nightly = version_meta().unwrap().channel {
+        println!("cargo::rustc-check-cfg=cfg(CHANNEL_NIGHTLY)");
+        println!("cargo:rustc-cfg=CHANNEL_NIGHTLY");
+    }
```

Comment:
It must be a matter of a separate PR.

Reply:
I tried `console` and even `crossterm` by just printing the key I pressed, they can recognize many keys modified by `Shift`, but `Enter`. 

The strange thing is, `Ctrl` `Alt` `Shift` work for many keys, but no matter which modifier I'm pressing, those crates just recognize `Enter` as long as I press it.

I'll take a look at `zed`